README.md

```diff
@@ -1,33 +1,94 @@
 # xk6-output-elasticsearch
-k6 extension for publishing test-run metrics to Elasticsearch.
 
-### Prerequisites
+[k6 extension](https://k6.io/docs/extensions/) for publishing test-run metrics to Elasticsearch.
 
-Go 1.17 or better (verify with `go version`).
+## Prerequisites
 
-### Getting started
+[Go](https://golang.org/) 1.17 or better (verify with `go version`).
 
-```
+## Install
+
+Install [xk6](https://k6.io/docs/extensions/guides/build-a-k6-binary-using-go/):
+
+```shell
+# Install xk6
+go install go.k6.io/xk6/cmd/xk6@latest
+
+# Build the xk6 binary locally
 git clone git@github.com:elastic/xk6-output-elasticsearch.git
-# build k6 with the Elasticsearch output extension
+# Build k6 locally with the Elasticsearch output extension
 make
 ```
 
+You will have a `k6` binary in the current directory.
+
+### Using Docker
+
+This [Dockerfile](./Dockerfile) builds a docker image with the k6 binary.
+
+## Configuration
+
+### Running on Elastic Cloud
+
 You can run the new k6 binary against a Cloud cluster with:
-```
+```shell
 export K6_ELASTICSEARCH_CLOUD_ID=your-cloud-id-here
 export K6_ELASTICSEARCH_USER=elastic
 export K6_ELASTICSEARCH_PASSWORD=your-password-here
 
 ./k6 run script.js -o output-elasticsearch
 ```
 
-Alternatively you can send metrics to a local cluster (without security, self-signed certificates are not supported yet):
+### Running a local cluster
 
-```
+Alternatively, you can send metrics to a local (unsecured) cluster:
+
+```shell
 export K6_ELASTICSEARCH_URL=http://localhost:9200
 
-./k6 run script.js -o output-elasticsearch
+./k6 run ./samples/script.js -o output-elasticsearch
+```
+
+If running locally with TLS (with a self-signed certificate), set `K6_ELASTICSEARCH_INSECURE_SKIP_VERIFY` to `true` (defaults to `false`):
+
+```shell
+export K6_ELASTICSEARCH_URL=https://localhost:9200
+export K6_ELASTICSEARCH_INSECURE_SKIP_VERIFY=true
+
+./k6 run ./samples/script.js -o output-elasticsearch
 ```
 
 The metrics are stored in the index `k6-metrics` which will be automatically created by this extension. See the [mapping](pkg/esoutput/mapping.json) for details.
+
+## Docker Compose
+
+This repo includes a [docker-compose.yml](./docker-compose.yml) file based on the [documentation](https://www.elastic.co/guide/en/elasticsearch/reference/8.7/docker.html#docker-file), that starts Elasticsearch and Kibana. It also adds a custom build of k6 having the `xk6-output-elasticsearch` extension. This is just a quick way to showcase the usage, not meant for production usage.
```

Comment:
Nit: Instead of referencing a specific version in the stack documentation link, can we instead use `current` so it resolves to the latest release?

Reply:
Change done, thanks! I tend to tie it to a specific version in case the documentation changes, but we can always update the links if they change.